File: Factors of Number/Factors of Number.cpp
#include<iostream>
using namespace std;

int main(){
    int num;
    cout<<"Please enter  number ==> ";
    cin>>num;
    cout<<"Factors of Given Number are ==> ";
    for(int i=1;i<=num;i++){
        if(num%i == 0){
            cout<<i<<", ";
        }
    }
    return 0;
}

File: Factors of Number/Factors of Number.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Tue Oct  6 18:25:31 2020

@author: ravi
"""

n = int(input("Please enter number ==> "))
for i in range(1,n+1):
    if n%i == 0:
        print(i,end = ",")